src/particles/ImpactXParticleContainer.H

```diff
@@ -52,11 +52,11 @@ namespace impactx
     {
         enum
         {
-            ux,  ///< momentum in x, normalized to proper velocity
-            uy,  ///< momentum in y, normalized to proper velocity
-            pt,  ///< kinetic energy deviation, (E0-E)/mc^2
-            m_qm, ///< charge to mass ratio, in q_e/m_e (q_e/eV)
-            w,   ///< particle weight, unitless
+            ux,  ///< momentum in x, normalized to reference longitudinal momentum (unitless)
+            uy,  ///< momentum in y, normalized to reference longitudinal momentum (unitless)
+            pt,  ///< kinetic energy deviation, normalized to (E0-E)/mc^2 (unitless)
```

Comment:
@cemitch99 in `t`, does `pt` still have the same normalization (energy deviation) or is actually a momentum deviation?

Reply:
Great question.  After the Z-T transformation, when the particle coordinates are obtained at fixed t, the component denoted "pt" is used to store the longitudinal momentum deviation pz.  (Likewise "t" is used to store the deviation in longitudinal position z.)  The stored value of pz is unitless.  I will double-check whether it is in static units (normalized by the reference momentum) or dynamic units (normalized by mc).

Reply:
Thanks!

We can probably use for these SoA attributes a similar doc syntax as we use for the SoA attributes:
https://impactx.readthedocs.io/en/latest/_static/doxyhtml/structimpactx_1_1_real_ao_s.html

Something like:
```
momentum in z, normalized to reference longitudinal momentum [unitless] (at fixed t) OR kinetic energy deviation, normalized to (E0-E)/mc^2 [unitless] (at fixed z) 
```

Reply:
Another random thought on nomenclature: why do we say "at fixed z" when we do mean "at fixed s"?

Reply:
Yes, right now "at fixed s" is more appropriate.  However, in the future some soft-edged elements may use a local "z" coordinate, so this is not so straightforward.  For the documentation, for now "at fixed s" works.

Reply:
Thanks! just to clarify,
> in the future some soft-edged elements may use a local "z" coordinate

does this influence the reference particle or just the "local" math we will do in that element?

Reply:
Just the "local" math.  The reference particle "z" argument has a fixed, global interpretation.

Reply:
> @cemitch99 in `t`, where `pt` is `pz` - does `pt` still have the same normalization (energy deviation) or is actually a momentum deviation, also normalized to reference longitudinal momentum (unitless)?

The current implementation of the Z-T (and T-Z) transformations use momenta normalized by mc.  It will be better if I modify this to work with either normalization via an input flag.  Then the interpretation will be clear.

Reply:
This is a minor change; I'll submit a PR soon.

Reply:
Does this proposed change look correct for now?
```suggestion
            ux,  ///< momentum in x, normalized to reference longitudinal momentum (unitless)
            uy,  ///< momentum in y, normalized to reference longitudinal momentum (unitless)
            pt,  ///< momentum in z, normalized to reference longitudinal momentum [unitless] (at fixed t) OR kinetic energy deviation, normalized to (E0-E)/mc^2 [unitless] (at fixed s)
```

Reply:
update: pt/pz are currently both normalized to (E0-E)/mc^2

Reply:
Line 57 needs to be modified.  pz is normalized to the magnitude of the reference momentum (call it, say, p0), in the same way as px and py.  However, pt is normalized to p0*c.  Explicitly, pt = (E0-E)/(p0*c).